java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -39,6 +43,10 @@ public abstract class AbstractJSONSchema2Pojo {
   static final String OBJECT_CRD_TYPE = "object";
   static final String ARRAY_CRD_TYPE = "array";
 
+  public static final AnnotationExpr GENERATED_ANNOTATION = new SingleMemberAnnotationExpr(
+      new Name("javax.annotation.processing.Generated"),
+      new StringLiteralExpr("io.fabric8.java-generator"));
```

Comment:
I think that the Generated annotation expects a fully qualified name of the Generator class (not sure what this might be in this context `CRGeneratorRunner.class.getName()` :shrug: )

Reply:
You are very right ... 🤷 
Done!